sdk/management/docs/AUTH.md

```diff
@@ -0,0 +1,92 @@
+# Authentication in Azure Management Library for Java
+
+To use the APIs in the Azure Management Library for Java, as the first step you need to create an authenticated client. This document is to introduce several possible approaches for authentication.
+
+## Prerequisites
+
+Before authenticating the Azure client, there are a few authentication classes you need to know before you decide which kind of authentication would meet your requirement.
+
+* TokenCredential
+  * The `TokenCredential` is an interface in the `azure-core` package for credentials that can provide a token. 
+  * Azure Identity offers multiple implementations of the `TokenCredential` class in the `azure-identity` package. To learn more, see [credentials in Azure Identity](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#credentials).
+
+Sample code to create a simple `ClientSecretCredential`:
+
+```java
+ClientSecretCredential clientSecretCredential = new ClientSecretCredentialBuilder()
```

Comment:
We might also need a sample for user to change auth endpoint for non-global cloud. Env var "AZURE_AUTHORITY_HOST", or `AadCredentialBuilderBase.authorityHost`.